docs/feature_split_keyboard.md

```diff
@@ -239,7 +244,7 @@ This sets the poll frequency when detecting master/slave when using `SPLIT_USB_D
 
 ## Hardware Considerations and Mods
 
-While most any Pro Micro can be used, micro controllers like the AVR Teensys and most (if not all) ARM boards require the Split USB Detect.
+While most any Pro Micro can be used without `SPLIT_USB_DETECT`, micro controllers like the AVR Teensys and most (if not all) ARM boards require that you define it (it is the default for ARM).
```

Comment:
It's not required to define it for ARM boards, it's automatically set, if it's a ChibiOS board.

Reply:
@drashna I rewrote the parts mentioning specific targets